src/components/Portfolio/PortfolioTokensSkeleton.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  AvatarGroup,
+  Badge,
+  Box,
+  Skeleton,
+  Stack,
+  useTheme,
+} from '@mui/material';
+import generateKey from '@/app/lib/generateKey';
+import { WalletCardContainer } from '../Menus';
+
+export default function PortfolioTokenSkeleton() {
+  const theme = useTheme();
+
+  return (
+    <>
+      {Array.from({ length: 10 }, () => 42).map((_, index) => (
+        <WalletCardContainer key={index} sx={{ paddingX: '16px !important' }}>
+          <Stack direction="row" spacing={2} sx={{ width: '100%' }}>
+            <Box>
+              <Badge
+                overlap="circular"
+                anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+                badgeContent={
+                  <Skeleton
+                    variant="circular"
+                    width={16}
+                    height={16}
+                    sx={{
+                      border: `2px solid ${theme.palette.surface2.main}`,
+                      backgroundColor:
+                        theme.palette.mode === 'light' ? '#e4e4e4' : '#3f3d56',
+                    }}
+                  />
+                }
+              >
+                <Skeleton variant="circular" width={40} height={40} />
+              </Badge>
+            </Box>
+            <Box
+              sx={{
+                width: '100%',
+                display: 'flex',
+                justifyContent: 'space-between',
+                alignContent: 'center',
+                alignItems: 'center',
```

Comment:
Looking at this quickly, sure I´d ask to put this into a styled-component.

But on top of that, I am wondering if it makes sense to come up with a "default container" that can be re-used? But maybe it´s over the top?? See, I found multiple components which share a similar setup:

![image](https://github.com/user-attachments/assets/1f5072c2-4498-4f23-bbbe-b8bd91599520)
![image](https://github.com/user-attachments/assets/bbe7b363-4357-4035-9a1b-699f7766e8d3)
![image](https://github.com/user-attachments/assets/19acbaaa-cc58-4408-8266-749f7ba3a908)
...sure there is more.

Wdyt? Is it any useful to create a "default" styled-component that we can inherit from and refactor?

Reply:
Ok I'll make a styled component. agree with you on the component. Will create a ticket.